src/main/java/org/sonarlint/cli/report/IssuesReport.java

```diff
@@ -85,11 +86,21 @@ public void setDate(Date date) {
     return new ArrayList<>(resourceReportsByFilePath.values());
   }
 
-  public List<String> getResourcesWithReport() {
+  public List<Path> getResourcesWithReport() {
     return new ArrayList<>(resourceReportsByFilePath.keySet());
   }
+  
+  public String getRuleName(String ruleKey) {
+    return ruleNameByKey.get(ruleKey);
+  }
+  
+  public String issueId(IssueListener.Issue issue) {
+    String line = issue.getStartLine() != null ? issue.getStartLine().toString() : "-";
+    return issue.getRuleKey() + "R" + issue.getFilePath().toString() + "L" + line;
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) NullPointerException might be thrown as 'issue' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2259)